install_ubuntu_gpu_drivers.sh

```diff
@@ -94,7 +94,15 @@ case $INSTALL_DRIVER_VERSION in \
         curl -L -O https://github.com/intel/compute-runtime/releases/download/24.26.30049.6/libigdgmm12_22.3.20_amd64.deb ; \
         curl -L -O https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.17193.4/intel-igc-core_1.0.17193.4_amd64.deb ; \
         curl -L -O https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.17193.4/intel-igc-opencl_1.0.17193.4_amd64.deb ; \
-
+        dpkg -i *.deb && rm -Rf /tmp/gpu_deps ; \
+;; \
+"24.39.31294") \
```

Comment:
That version list has grown quite significantly. Maybe we could remove older versions? Do we still need option to install 21.48?